plugins/discovery/discovery.go

```diff
@@ -354,6 +378,27 @@ func getPluginSet(factories map[string]plugins.Factory, manager *plugins.Manager
 		return nil, err
 	}
 
+	// If the discovery plugin is configured to use the manual triggering mode, then check that the bundle, decision log
```

Comment:
This seems like behaviour that we should document as it's not going to be immediately obvious without trying it out first.

Reply:
Added a note about this in the doc.